src/analysis/retail/hunter/survival/modules/talents/FrenzyStrikes.tsx

```diff
@@ -54,28 +53,28 @@ class Butchery extends Analyzer {
     if (this.reductionAtCurrentCast === BUTCHERY_CARVE_MAX_TARGETS_HIT) {
       return;
     }
-    this.reductionAtCurrentCast += 1;
+    this.reductionAtCurrentCast += 3;
     if (this.spellUsable.isOnCooldown(TALENTS.WILDFIRE_BOMB_TALENT.id)) {
       this.checkCooldown(TALENTS.WILDFIRE_BOMB_TALENT.id);
     } else {
-      this.wastedReductionMs += ONE_SECOND_IN_MS;
+      this.wastedReductionMs += 3000;
     }
   }
 
   checkCooldown(spellId: number) {
-    if (this.spellUsable.cooldownRemaining(spellId) < ONE_SECOND_IN_MS) {
-      const effectiveReductionMs = this.spellUsable.reduceCooldown(spellId, ONE_SECOND_IN_MS);
+    if (this.spellUsable.cooldownRemaining(spellId) < 3000) {
```

Comment:
I'd recommend making this `3000` a constant, e.g. `FRENZY_STRIKES_CDR` or something similar